sidechain/sidechain_block_verification/Cargo.toml

```diff
@@ -0,0 +1,51 @@
+[package]
+name = "sidechain_block_verification"
+description = "Verification logic for sidechain blocks"
+version = "0.9.0"
+authors = ["Integritee AG"]
+homepage = "https://integritee.network/"
+repository = "https://github.com/integritee-network/pallets/"
+license = "Apache-2.0"
+edition = "2021"
+
+
+[dependencies]
+log = { version = "0.4.17", default-features = false }
+thiserror = { version = "1.0.26", optional = true }
+
+# local deps
+itp-types = { path = "../../primitives/types", default-features = false }
+itp-utils = { path = "../../primitives/utils", default-features = false }
+sidechain-primitives = { path = "../../primitives/sidechain", default-features = false }
+
+# substrate deps
+frame-support = { default-features = false, git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.24" }
+sp-core = { default-features = false, git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.24" }
+sp-consensus-slots = { default-features = false, git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.24" }
+sp-runtime = { default-features = false, git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.24" }
+
+# sgx deps
+thiserror-sgx = { package = "thiserror", git = "https://github.com/mesalock-linux/thiserror-sgx", tag = "sgx_1.1.3", optional = true }
+sgx_tstd = { branch = "master", features = ["untrusted_fs", "net", "backtrace"], git = "https://github.com/apache/teaclave-sgx-sdk.git", optional = true }
+
+[features]
+default = ["std"]
+std = [
+    "log/std",
+    "thiserror",
+    # local
+    "itp-types/std",
+    "itp-utils/std",
+    "sidechain-primitives/std",
+    # substrate
+    "sp-runtime/std",
+]
+sgx = [
+    "sgx_tstd",
+    "thiserror-sgx",
+]
+
+[dev-dependencies]
+parentchain-test = { path = "../../parentchain/test", default-features = false }
```

Comment:
Remove the `default-features = false` here. Cargo test is always `std`, so it's safe to remove.